test/screenshot/webpack-bundles.js

```diff
@@ -42,8 +47,14 @@ module.exports.bundle = function(testPath, outputPath) {
       }],
     },
     plugins: [
-      new ExtractTextPlugin(outputPath + '.css'),
+      new ExtractTextPlugin('bundle.css'),
+      new HtmlWebpackPlugin({
+        template: path.resolve(__dirname, 'index.hbs'),
+      }),
       new OptimizeCssAssetsPlugin(),
     ],
+    devServer: {
+      historyApiFallback: true,
```

Comment:
but why?

Reply:
This is how the dev server knows to serve the single page app again. Say we go to `temp-package/foo` path. That is not an html file, so we serve the index.html file, which hits the JS bundle. The react router then routes to the correct JS to serve based on the location.